src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,80 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetEventProblemsParams extends OrganizationParamParts, TimeRangeParamParts, Paginated {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType?: string[];
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub?: string[];
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId?: string[];
+}
+export interface EventProblemsResponse extends PaginatedResponse {
+    /**
+     * List of the event problems.
+     */
+    problems: EventProblemsResponseItem[];
+}
+
+export interface EventProblemsResponseItem {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType: string;
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId: string;
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub: string;
+    /**
+     * Number of failed events.
+     */
+    failedEvents: number;
+    /**
+     * Number of total events.
+     */
+    totalEvents: number;
+    /**
+     * List of the validation errors.
+     */
+    validationErrors: ValidationErrors[];
+    /**
+     * Events sampled from the problematic events.
+     */
+    sampleEvents: string[];
+    /**
+     * Timestamps from the sample events.
```

Comment:
I presume this relates to the `sampleEvents`.

It makes me wonder why the sample events are not returned as a `{ sampleEvent: string; timestamp: string }[]`.

Reply:
It's the result of a design decision made a while back, it's just how the API works.